.circleci/config.yml

```
@@ -5,29 +5,69 @@ orbs:
   win: circleci/windows@1.0.0
   docker: circleci/docker@0.5.20
 
-jobs:
-  vet:
+executors:
+  base-linux:
     docker:
-      - image: diane/gggv-base:latest
-    environment:
-      CGO_CFLAGS: -w
+      - image: circleci/golang:1.13
     working_directory: /go/src/github.com/dianelooney/gggv
+
+commands:
+  install-deps:
     steps:
       - checkout
+      - restore_cache:
+          keys:
+            - v1-pkg-cache
+      - run: sudo ./build/gggv/configure.sh
       - run: go get -v -t -d ./...
+      - save_cache:
+          key: v1-pkg-cache
+          paths:
+            - /go/pkg
+
+  install-deps-windows:
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: choco install mingw pkgconfiglite
+      - run:
+          name: Setup pkg-config stubs
+          command: ./build/windows/configure.sh
+      # - run:
+      #     name: Install Portaudio
+      #     command: |
+      #       curl http://www.portaudio.com/archives/pa_stable_v190600_20161030.tgz > /tmp/portaudio.tgz
+      #       7z x /tmp/portaudio.tgz -o/tmp/
+      #       7z x /tmp/portaudio.tar -o/tmp/
+      #       cd /tmp/portaudio && ./configure && make install
+      - run:
+          name: Install FFMPEG-dev
+          command: |
+            curl https://ffmpeg.zeranoe.com/builds/win64/dev/ffmpeg-latest-win64-dev.zip > /tmp/ffmpeg-latest-win64-dev.zip
+            unzip /tmp/ffmpeg-latest-win64-dev.zip -d /tmp/
+      - run:
+          name: Install go 1.13
+          command: choco install golang
```

```suggestion
          command: choco install golang --force
```
Need to fix this at some point:
```
#!bash.exe
choco install golang
Chocolatey v0.10.15
Installing the following packages:
golang
By installing you accept licenses for the packages.
golang v1.12.7 already installed.
 Use --force to reinstall, specify a version to install, or try upgrade.

Chocolatey installed 0/1 packages. 
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Warnings:
 - golang - golang v1.12.7 already installed.
 Use --force to reinstall, specify a version to install, or try upgrade.
CircleCI received exit code 0
```